<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresetsList</class>
 <widget class="QWidget" name="PresetsList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="HelperLibraryNS::ExtList" name="extList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="saveBtn">
       <property name="toolTip">
        <string>Сохранить предустановку</string>
       </property>
       <property name="statusTip">
        <string>Сохранить предустановку</string>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeBtn">
       <property name="toolTip">
        <string>Удалить выбранную предустановку</string>
       </property>
       <property name="statusTip">
        <string>Удалить выбранную предустановку</string>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="loadBtn">
       <property name="toolTip">
        <string>Загрузить выбранную предустановку</string>
       </property>
       <property name="statusTip">
        <string>Загрузить выбранную предустановку</string>
       </property>
       <property name="text">
        <string>Загрузить</string>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HelperLibraryNS::ExtList</class>
   <extends>QWidget</extends>
   <header>HelperLibrary/Widgets/ListWidgets/extlist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
